.travis.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - TEST_TYPE=lint
     script:
       - make check 2>&1
-      - go install ./vendor/honnef.co/go/tools/cmd/staticcheck
+      - go get honnef.co/go/tools/cmd/staticcheck
```

Comment:
we use frozen version of staticcheck, please update

Reply:
done